client/.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+.env
```

Comment:
Move this file to the root.

Please hide your `.idea` and `.vscode` folder.  Please hide the `app.sqlite` file as well.